### DISCUSSION TITLE
Cannot set "white-space: pre-line" only

### DISCUSSION THREAD

**Discussion by dogukanarig** [quasarframework/quasar#15938] [General - Components / Directives / etc]
I created a custom CSS class for q-item-label. I want to show the spaces in the text with this class, but white-space: pre-line is not working as intended. Can you please help me?

`        

        <q-item-section>
          <q-item-label class="text-subtitle1">
            <strong>Janet</strong>
            <span class="text-grey-7">
              @janet_Marcel
            </span>
          </q-item-label>

          <q-item-label class="text-body1 qweet-content">
            Lorem ipsum dolor
            sit amet

            consectetur adipisicing elit. Ex ut consectetur voluptas sunt odit maxime,
            architecto
            enim iusto harum non.
          </q-item-label>
        </q-item-section>`

`<style lang="sass">
.qweet-content
  white-space: pre-line
</style>`

**Comment by suleimanas** [comment]
Can you please make a codepen
